docroot/modules/custom/sitenow_articles/src/Form/SettingsForm.php

```diff
@@ -185,6 +185,24 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#size' => 60,
     ];
 
+    $form['global']['sitenow_articles_default_featured_image_size'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Articles Sitewide Default Featured Image Size'),
+      '#description' => $this->t('Sets the Default Featured Image Size for Articles'),
```

Comment:
Let's change this to 'Set the default behavior for how to display a featured image.'